src/providers/create/ProvidersCreatePage.tsx

```diff
@@ -0,0 +1,150 @@
+import { type FC, useState } from 'react';
+import { useHistory } from 'react-router';
+import { encode } from 'js-base64';
+import { useK8sWatchProviderNames } from 'src/modules/Providers/hooks/useK8sWatchProviderNames';
+import useToggle from 'src/modules/Providers/hooks/useToggle';
+import { Namespace } from 'src/utils/constants';
+import { useForkliftTranslation } from 'src/utils/i18n';
+import { getDefaultNamespace } from 'src/utils/namespaces';
+
+import type { IoK8sApiCoreV1Secret, V1beta1Provider } from '@kubev2v/types';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import { Divider, Form, PageSection } from '@patternfly/react-core';
+
+import { getProviderDetailsPageUrl } from '../utils/getProviderDetailsPageUrl';
+import { type ValidationMsg, ValidationState } from '../utils/types';
+import { validateProjectName } from '../utils/validators/validateProjectName';
+import { validateProviderAndSecret } from '../utils/validators/validateProviderAndSecret';
+
+import ProviderCreateActionsSection from './components/ProviderCreateActionsSection';
+import ProvidersCreateFormsSection from './components/ProvidersCreateFormsSection';
+import ProvidersCreatePageHeader from './components/ProvidersCreatePageHeader';
+import { createProvider } from './utils/createProvider';
+import { createProviderSecret } from './utils/createProviderSecret';
+import { patchProviderSecretOwner } from './utils/patchProviderSecretOwner';
+import { getProviderTemplateWithNamespace } from './utils/providerTemplates';
+import { getSecretTemplateWithNamespace } from './utils/secretTemplates';
+
+import './ProvidersCreatePage.style.scss';
+
+const ProvidersCreatePage: FC<{
+  namespace: string;
+}> = ({ namespace }) => {
+  const { t } = useForkliftTranslation();
+  const history = useHistory();
+  const [isLoading, toggleIsLoading] = useToggle();
+  const [activeNamespace, setActiveNamespace] = useActiveNamespace();
+  const [providerNames, providerNamesLoaded] = useK8sWatchProviderNames({ namespace });
+
+  const defaultNamespace = getDefaultNamespace();
+  const initialProjectName =
+    activeNamespace === Namespace.AllProjects ? defaultNamespace : activeNamespace;
+  const initialNamespace = namespace || initialProjectName || defaultNamespace;
```

Comment:
Why we have namespace as a prop, we get the active namespace as well we also get the default namespace, initialProjectName and initialNamespace to capitalize them all.
This is too much constants for namespace context, please check if the prop namespace and the activeNamespace are the same I believe they should be, and please simplify this, there should be just one constant for the namespace eventually to have one source of truth